src/item.rs

```diff
@@ -631,10 +630,10 @@ impl Printer {
             self.type_param_bound(&bound);
         }
         if let Some((_eq_token, default)) = &trait_item.default {
-            self.where_clause_oneline(&trait_item.generics.where_clause);
-            self.word("= ");
+            self.word(" = ");
             self.neverbreak();
             self.ty(default);
+            self.where_clause_oneline_semi(&trait_item.generics.where_clause);
```

Comment:
Please move this line out of the `if` block since it is identical to the other branch.

Reply:
Done.